src/plugins/wizard/public/application/utils/drag_drop/types.ts

```diff
@@ -3,15 +3,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { IndexPatternField } from '../../../../../data/common';
+import { IndexPatternField, METRIC_TYPES } from '../../../../../data/common';
+
+export const COUNT_FIELD = Symbol.for(METRIC_TYPES.COUNT);
 
 export interface EmptyDragDataType {
   namespace: null;
   value: null;
 }
 export interface FieldDragDataType {
   namespace: 'field-data';
-  value: Pick<IndexPatternField, 'name' | 'displayName' | 'type'> | null;
+  value: IndexPatternField['name'] | null | typeof COUNT_FIELD;
```

Comment:
nit - I can't particularly justify why, but I think it reads nicer if `null` is last in the union.

Reply:
Will update it in a follow-up PR